sdk-scheme-adapter/v2_0_0/components/parameters/transactionRequestId.yaml

```diff
@@ -0,0 +1,8 @@
+name: transactionRequestId
+in: path
+required: true
+schema:
+  $ref: ../schemas/CorrelationId.yaml
+description: >-
+  Identifier of the merchant request to pay to continue as returned in
+  the response to a `POST /requestToPay` request.
```

Comment:
What about all the other API operations?

And how does it link to the FSPIOP spec?

Reply:
Currently there are no cross references in the api-snippets. So I just followed the convention.